docs/UserGuide.md

```diff
@@ -59,13 +59,16 @@ Coming soon...
 
 * If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
   </box>
+<<<<<<< HEAD
```

Comment:
Thanks for editing the UG! Might want to remove this merge conflict messages hahaha